File: Programmers/Lv0/잘라서배열로저장하기.py
def solution(my_str, n):
    answer = []
    start_idx = 0
    end_idx = n
    while True:
        if len(my_str) <= end_idx:
            break
        else:
            answer.append(my_str[start_idx:end_idx])
            start_idx += n
            end_idx += n
    answer.append(my_str[start_idx:])
    return answer

File: Programmers/Lv0/잘라서배열로저장하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(string my_str, int n) {
    vector<string> answer;
    size_t start = 0;
    while (1) {
        if (my_str.length() <= start + n)
            break ;
        else {
            answer.push_back(my_str.substr(start, n));
            start += n;
        }
    }
    answer.push_back(my_str.substr(start, my_str.length() - start));
    return answer;
}